public/app/features/alerting/unified/components/contact-points/utils.ts

```diff
@@ -152,6 +152,8 @@ export function enhanceContactPointsWithMetadata(
       }),
     };
   });
+
+  return enhanced.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Moved sorting into here so it's always done for us when we "enhance" the contact points